docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -376,7 +376,7 @@
         </tr>
         <tr>
             <td><h5>write-mode</h5></td>
-            <td style="word-wrap: break-word;">change-log</td>
+            <td style="word-wrap: break-word;">append-only</td>
```

Comment:
We may need a `auto` write mode...
Append-only for non-primary-key.
Change-log for primary key.

Reply:
@JingsongLi Thanks. done.